lib/cocoapods/target/pod_target.rb

```diff
@@ -594,21 +563,5 @@ def uses_modular_headers?(only_if_defines_modules = true)
       return false if only_if_defines_modules && !defines_module?
       spec_consumers.none?(&:header_mappings_dir) && spec_consumers.none?(&:header_dir)
     end
-
-    private
-
-    # @param  [TargetDefinition] target_definition
-    #         The target definition to check.
-    #
-    # @return [Array<Dependency>] The dependency of the target definition for
-    #         this Pod. Return an empty array if the Pod is not a direct
-    #         dependency of the target definition but the dependency of one or
-    #         more Pods.
-    #
-    def target_definition_dependencies(target_definition)
-      target_definition.dependencies.select do |dependency|
```

Comment:
This is the slow bit that's being removed -- `target_definition.dependencies` isn't cheap